src/main/kotlin/no/nav/helse/ArbeidsforholdOppslag.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.helse
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.github.kittinunf.fuel.httpGet
+import no.nav.helse.serde.defaultObjectMapper
+import java.time.LocalDate
+
+class ArbeidsforholdOppslag(val sparkelUrl: String, val stsRestClient: StsRestClient) {
+
+    fun hentArbeidsforhold(sykepengesoknad: Sykepengesoknad) : ArbeidsforholdFakta {
+        val arbeidsforhold = hentArbeidsforholdRest(AktorId(sykepengesoknad.aktorId), sykepengesoknad.fom, sykepengesoknad.tom)
```

Comment:
Jeg vet ikke hvilken fom/tom du burde bruke, men jeg tror ikke det er sykepengesøknadens. Den fom/tom gjelder perioden brukeren søker penger for, mens vi er interessert i å finne ut av hva vedkommende har hatt av arbeidsforhold _før_ sykdommen.

Mulig det bør være noe sånt som tom = søknad.førsteSykdomsdag og fom = søknad.førsteSykdomsdag minus et par måneder

Reply:
den er god 👍